scheduler.py

```
@@ -240,6 +243,122 @@ async def _send_medicine_reminder(self, user_id: int, medicine_id: int):
+                return
+
+            message = f"{config.EMOJIS['reminder']} *תזכורת*\n\n{reminder.text}"
+            await self.bot.send_message(chat_id=user.telegram_id, text=message, parse_mode="Markdown")
+
+            if reminder.repeat == "once":
+                # Nothing left to fire; keep the list clean without deleting history
+                await DatabaseManager.set_custom_reminder_active(reminder_id, False)
+
+            logger.info(f"Sent custom reminder {reminder_id} to user {user_id}")
+
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)